app/models/scimitar/resources/base.rb

```diff
@@ -137,18 +137,21 @@ def constantize_complex_types(hash)
         end
       end
 
+      # Renders *in full* as JSON; typically used for write-based operations...
+      #
+      #     record = self.storage_class().new
+      #     record.from_scim!(scim_hash: scim_resource.as_json())
+      #     self.save!(record)
+      #
+      # ...so all fields, even those marked "returned: false", are included.
+      # Use Scimitar::Resources::Mixin::to_scim to obtain a SCIM object with
+      # returnable fields omitted, rendering *that* as JSON via #to_json.
```

Comment:
This is stil somewhat clear as mud, but at least it gives a fairly strong indication that "you probably don't want to call this for returned JSON data". Between this and the clear examples of controller code in `README.md`, hopefully it's enough for now.